pydm/data_plugins/epics_plugins/pva_codec.py

```diff
@@ -0,0 +1,108 @@
+import io
+import logging
+import numpy as np
+from functools import reduce
+from p4p.wrapper import Value
+
+logger = logging.getLogger(__name__)
+
+
+ScalarType = (bool, np.int8, np.int16, np.int32, np.compat.long, np.uint8,
+              np.uint16, np.uint32, np.uint64, float, np.double)
+
+
+codecs = {}
+
+
+def decompress(structure: Value):
```

Comment:
Lots of decompression code here to test out. Ideally this would come from an IOC on docker run during CI, but that can be a real pain.

Maybe instead - in order to get some coverage here: configure an IOC of your own outside of the PyDM repo, reconfigure the areadetector with a variety of compression schemes, dump out small sample values, and make sure your `decompress()` function can handle each of them?

Reply:
Created a separate issue for writing these tests at #884